app/controllers.js

```diff
@@ -7,9 +7,19 @@ const getTopics = (request, response, next) => {
 };
 
 const getArticles = (request, response, next) => {
-        fetchArticles().then((articles) => {
-            response.status(200).send({ articles: articles });       
-        });
+    fetchArticles().then((articles) => {
+        response.status(200).send({ articles: articles });       
```

Comment:
You could shorten this to just {articles} if you wanted (it's shorthand for the same thing!)